lib/fathead/django_built_ins/parse.py

```diff
@@ -0,0 +1,177 @@
+from bs4 import BeautifulSoup
+
+DJANGO_HOME = 'https://www.djangoproject.com/'
+DJANGO_DOC_URL = 'https://docs.djangoproject.com/en/1.9/ref/templates/builtins/'
```

Comment:
@milmazz any suggestions for keeping these in sync? My thought was to have fetch.sh just call a different bash script that allows for arguments. So fetch.sh would be something like:
```
#!/bin/bash
./fetch_html.sh 1.9
```

This still would mean two places to update version numbering. Maybe we could have a text file that just contains the version number and pass it to each script? Thoughts?